content/en/docs/reference/kubernetes-api/ref/common-definitions/typedlocalobjectreference-.md

```diff
@@ -0,0 +1,39 @@
+---
+api_metadata:
+  apiVersion: ""
```

Comment:
This file is a candidate for renaming with hyphens:
`typed-local-object-reference-v1.md`